zebra-chain/src/orchard/keys.rs

```diff
@@ -60,6 +59,9 @@ fn prp_d(K: [u8; 32], d: [u8; 11]) -> [u8; 11] {
 // TODO: This is basically a duplicate of the one in our sapling module, its
 // definition in the draft Nu5 spec is incomplete so I'm putting it here in case
 // it changes.
+//
+// Review question: @dconnolly is this code still incomplete?
+// It's used to generate new note commitments from notes.
```

Comment:
@dconnolly can you check if this code is still incomplete?

Reply:
I'd still like a cryptographer to review this code, it looks like it might be used in production.

Reply:
The code looks correct regarding the spec, but I don't think it's used anywhere. `NoteCommitment::new` is not used anywhere. Ultimately this `prf_expand` is used to implement `Arbitrary` for orchard `TransmissionKey` and `Address` which don't seem to be required currently.

Reply:
I think you're right!

Originally I thought we used it to generate arbitrary Orchard addresses, notes, note commitments, and transactions. But it turns out our EncryptedNote and WrappedNote key generation just uses random bytes.

Do you think it's worth removing it now? I don't mind either way.